scripts/generate-readme-table.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+const rules = require("../lib/rules");
+
+const README_LOCATION = path.resolve(__dirname, "..", "readme.md");
+const BEGIN_TABLE_MARKER = "<!-- __BEGIN AUTOGENERATED TABLE__ -->\n";
+const END_TABLE_MARKER = "\n<!-- __END AUTOGENERATED TABLE__ -->";
+
+const expectedTableLines = Object.keys(rules)
+    .sort()
+    .reduce((lines, ruleId) => {
+        const rule = rules[ruleId];
+
+        lines.push([
+            `[${ruleId}](https:/eslint.org/docs/rules/${ruleId})`,
+            "🛠",
+            rule.meta.docs.description
+        ].join(" | "));
+
+        return lines;
+    }, ["Name | 🛠 | Description", "----- | ----- | -----"])
+    .join("\n");
+
+const readmeContents = fs.readFileSync(README_LOCATION, "utf8");
+
+if (!readmeContents.includes(BEGIN_TABLE_MARKER)) {
+    throw new Error(`Could not find '${BEGIN_TABLE_MARKER}' marker in README.md.`);
```

Comment:
Put the marker into the error message will make the message messy, because the marker contains linebreak.

Reply:
maybe. but it should not happen as I have put the marker in readme.

Reply:
Of course, so whether does this check or not depends on you.